.idea/.gitignore

```diff
@@ -0,0 +1,30 @@
+# Default ignored files
```

Comment:
Add .idea folder to gitignore and remove from repo
Helpful link
https://stackoverflow.com/questions/11124053/accidentally-committed-idea-directory-files-into-git